runtime/astar/src/xcm_config.rs

```diff
@@ -274,6 +274,7 @@ impl xcm_executor::Config for XcmConfig {
     type CallDispatcher = WithOriginFilter<SafeCallFilter>;
     type SafeCallFilter = SafeCallFilter;
     type Aliasers = Nothing;
+    type TransactionalProcessor = ();
```

Comment:
Question - is there a specific reason why we're not using `FrameTransactionalProcessor` here?

Reply:
hmm, I don't see any reason not to. I might have missed this as well

Reply:
done